docs/src/content/docs/drivers/postgres/getting-started.mdx

```diff
@@ -0,0 +1,216 @@
+        const oid = typeToOid[jsType];
+        if (oid) {
+          transformedEntry[key] = sql.typed(o[key], oid);
+        } else {
+          throw new Error(`Unsupported JS type: ${jsType} for key: ${key}`);
+        }
+      }
+    }
+    return transformedEntry;
+  });
+}
+
+async function insertTrades(trades) {
+  if (!Array.isArray(trades) || trades.length === 0) {
+    throw new Error('Trades must be a non-empty array');
+  }
+
+  return await sql`
+    INSERT INTO trades
+    ${ sql(objsToTypedObjs(trades)) }
+  `;}
+
+async function getTradesOver(quantity) {
+  return await sql`
+    SELECT *
+    FROM trades
+    WHERE quantity > ${ sql.typed(quantity, 23) }
+  `;}
+
+async function main() {
+  const trades = [
+    { _id: 1, name: 'Trade1', quantity: 1001 },
+    { _id: 2, name: 'Trade2', quantity: 15 },
+    { _id: 3, name: 'Trade3', quantity: 200 },
+  ];
+
+  try {
+    const tx = await insertTrades(trades);
+    console.log('Trades inserted successfully');
+  } catch (error) {
+    console.error('Error inserting trades:', error);
+  }
+
+  try {
+    const result = await getTradesOver(100);
+    console.log(result);
+  } catch (error) {
+    console.error('Error fetching trades:', error);
+  }
+  process.exit()
+}
+
+main();
+```
+    </TabItem>
+    <TabItem label="Python">
+This example uses the [Psycopg 3](https://www.psycopg.org/psycopg3/) database adapter.
+
+```python
+import asyncio
+import psycopg as pg
+
+DB_PARAMS = {
+    "host": "localhost",
+    "port": 5432
+}
+
+async def insert_trades(conn, trades):
+    if not isinstance(trades, list) or len(trades) == 0:
+        raise ValueError("Trades must be a non-empty list")
+
+    query = """
+    INSERT INTO trades (_id, name, quantity) VALUES (%s, %s, %s)
+    """
+
+    trade_values = [(trade["_id"], trade["name"], trade["quantity"]) for trade in trades]
+
+    async with conn.cursor() as cur:
+        await cur.executemany(query, trade_values)
+
+async def get_trades_over(conn, quantity):
+    query = """
+    SELECT * FROM trades WHERE quantity > %s
+    """
+    async with conn.cursor() as cur:
+        await cur.execute(query, (quantity,))
+        return await cur.fetchall()
+
+async def main():
+    trades = [
+        {"_id": 1, "name": "Trade1", "quantity": 1001},
+        {"_id": 2, "name": "Trade2", "quantity": 15},
+        {"_id": 3, "name": "Trade3", "quantity": 200},
+    ]
+
+    try:
+        async with await pg.AsyncConnection.connect(**DB_PARAMS,
+                                                    autocommit=True) as conn:
+            conn.adapters.register_dumper(str, pg.types.string.StrDumperVarchar)
+            await insert_trades(conn, trades)
+            print("Trades inserted successfully")
+
+            result = await get_trades_over(conn, 100)
+            print(result)
+    except Exception as error:
+        print(f"Error occurred: {error}")
+
+if __name__ == "__main__":
+    asyncio.run(main())
+```
+    </TabItem>
```

Comment:
Would it be too cheeky to have a "More coming soon..." tab?
Maybe with a link to how to help out?